### ISSUE TITLE
Cannot set property crypto of #<Object> which has only a getter

### ISSUE THREAD

**Issue by iulh** [IS2511/ChatIS#31]
```
An unhandled error occured: GET /: TypeError: Cannot set property crypto of #<Object> which has only a getter
    at Function.assign (<anonymous>)
    at serve (file:///E:/Projects/node.js/ChatIS/node_modules/solid-start-cloudflare-pages/index.js:65:20)   
    at Object.fetch (<script>:4:26)
    at E:\Projects\node.js\ChatIS\node_modules\@miniflare\core\dist\src\index.js:1533:19
    at E:\Projects\node.js\ChatIS\node_modules\@miniflare\shared\dist\src\index.js:415:9
    at ServiceWorkerGlobalScope.<anonymous> (E:\Projects\node.js\ChatIS\node_modules\@miniflare\shared\dist\src\index.js:438:9)
    at [nodejs.internal.kHybridDispatch] (node:internal/event_target:820:20)
    at ServiceWorkerGlobalScope.dispatchEvent (node:internal/event_target:755:26)
    at ServiceWorkerGlobalScope.dispatchEvent (E:\Projects\node.js\ChatIS\node_modules\@miniflare\shared\dist\src\index.js:430:18)
    at ServiceWorkerGlobalScope.dispatchEvent (E:\Projects\node.js\ChatIS\node_modules\@miniflare\shared\dist\src\index.js:447:26)
```
i've just copied the repo, installed dependencies and ran `npm run dev`, no changes at all

**Comment by iiulh** [comment]
oh it might be because of cyrilic symbols i think

**Comment by iulh** [comment]
it's prob-ly browser support reasons, case closed

**Comment by iulh** [comment]
> it's prob-ly browser support reasons, case closed

proof checked this, and nope it's still the same (why would it be even if it's server-side)

**Comment by IS2511** [comment]
No idea what that error is, never seen it.
This is very much a WIP project, not deployed anywhere yet. I was drafting a design for the v3 homepage, doubt I'll use this code in any way later, but will use the visuals to build something similar from scratch (maybe with htmx...).
The currently deployed version is v2, available in [this](https://github.com/IS2511/ChatIS-v2) repo.

TLDR: you can't use this project, it's not ready yet.

**Comment by iulh** [comment]
> TLDR: you can't use this project, it's not ready yet.

do i come back later? good luck with the project and projects anyway
